lib/beautiful/xresources.lua

```diff
@@ -64,6 +64,9 @@ function xresources.get_current_theme()
 end
 
 
+--- Store the cached or user-set per-screen DPI
+-- Values are tables with key dpi (the dpi) and set_from (user, auto, xrdb)
+-- The same is done for xresources.dpi
```

Comment:
What this commit actually does is to add a cache to DPI values. Why? Isn't the existing state of recomputing this "all the time" good enough? Do you have some example where adding a cache here speeds something up significantly?